x-pack/plugins/fleet/server/services/security/uninstall_token_service/index.ts

```diff
@@ -226,7 +230,9 @@ export class UninstallTokenService implements UninstallTokenServiceInterface {
     const uninstallTokens: UninstallToken[] = tokenObject.map(
       ({ id: _id, attributes, created_at: createdAt, error }) => {
         if (error) {
-          throw new UninstallTokenError(`Error when reading Uninstall Token: ${error.message}`);
+          const errorMessage = 'Error when reading Uninstall Token';
+          appContextService.getLogger().error(`${errorMessage} with id '${_id}': '${error}'`);
```

Comment:
Since this logs the original error, it still has the potential to contain encrypted data. We should just remove `error` from the log completely.

Reply:
yeah, you're right, thanks!
8b9faa7cd0ad025ace0aff1427e305b42d3afe06